### DISCUSSION TITLE
Not sure about the difference of pnpm deploy and turbo prune

### DISCUSSION THREAD

**Discussion by ScreamZ** [vercel/turborepo#4492] [Help] [answered]
Hello,

over the last month i've saw that pnpm deploy command and turbo prune have been released, both seems to achieve the same goal for deploying docker image, so my question is : do I need to combine both or not to deploy a backend nodeJS app using pnpm workspace inside turborepo ?

Thanks

**Comment by anthonyshew** [comment, marked as answer]
`pnpm` and Turborepo appear to have some overlap in their functionality in this case. You would want to choose based on an evaluation of your requirements and needs.

Speaking for Turborepo, you can check out how to deploy Docker images on this doc: https://turbo.build/repo/docs/handbook/deploying-with-docker
